ratings/ratings.py

```diff
@@ -295,7 +295,7 @@ def wins_subset(bucket):
             "select model_winner, model_loser from wins "
             "join models where "
             "    models.bucket = ? AND "
-            "    (model_winner = models.id or model_loser = models.id)",
+            "    model_winner = models.id",
```

Comment:
use triple quoted string here to avoid bugs with trailing commas? SQL isn't space-sensitive.

Reply:
not sure what you mean -- there are no trailing commas needed?

Reply:
The last comma threw me off somewhat. Had to stare at the diff for 15 seconds to see what was going on.